client/components/add-beer-view/add-beer-view.jsx

```diff
@@ -107,8 +107,11 @@ class AddBeerView extends React.Component {
       bar !== DEFAULT_BAR
       && beerType !== DEFAULT_BEER_TYPE
       && volume !== DEFAULT_VOLUME
-      && abv > 0
-      && price > 0);
+      && abv > 0 && abv <= MAX_ABV
+      && price > 0) && price <= MAX_PRICE;
+    
+    const abvHint = abv >= 12 ? true : false;
+    const priceHint = price >= 30 ? true : false;
```

Comment:
just `const abvHint = abv >= 12` and `const priceHint = price >= 30` are enough

Reply:
Ah yes